<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditarFuncionario</class>
 <widget class="QDialog" name="EditarFuncionario">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>788</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Editar Funcionário</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(255,255,255);</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>710</y>
     <width>541</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnSalvarEdicaoFuncionario">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>   Salvar</string>
      </property>
      <property name="icon">
       <iconset resource="imagens.qrc">
        <normaloff>:/Imagens/Imagens/salvar.png</normaloff>:/Imagens/Imagens/salvar.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnCancelarEdicaoFuncionario">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>   Cancelar</string>
      </property>
      <property name="icon">
       <iconset resource="imagens.qrc">
        <normaloff>:/Imagens/Imagens/cancelar.png</normaloff>:/Imagens/Imagens/cancelar.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>541</width>
     <height>661</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblEditarFuncionario_ID">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>CPF:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtEditarFuncionario_CPF">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Nome:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtEditarFuncionario_Nome">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Salário:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtEditarFuncionario_Salario">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Departamento:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cmbEditarFuncionario_Departamento">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Administrativo</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Comercial</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Financeiro</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Jurídico</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Marketing</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Operacional</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Recursos Humanos</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Data de Nascimento:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtEditarFuncionario_DataNascimento">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Telefone:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtEditarFuncionario_Telefone">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Email:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtEditarFuncionario_Email">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="imagens.qrc"/>
 </resources>
 <connections/>
</ui>
